scripts/updateCLIDoc.js

```diff
@@ -0,0 +1,208 @@
+const exec = require('child_process').execSync;
+
+const fs = require('fs');
+const Handlebars = require('handlebars');
+
+const source = fs.readFileSync('./scripts/resources/CLITemplate.md', 'utf8');
+const template = Handlebars.compile(source);
+
+
+let first = true;
+let obj = {
+    common: "",
```

Comment:
The other scripts use tabs instead of spaces. Please this file to tabs as well.

This should actually be enforced automatically by the `.editorconfig` in the root directory, but maybe you are missing the corresponding extension or IDE configuration?